spec/support/command.rb

```diff
@@ -0,0 +1,18 @@
+def simple_command(v, opts = {})
```

Comment:
You shoudn't define method in top-level. Since the previous code was in describe block, it was defined to anonymous class created by RSpec.

It's better to define these methods in module and include it.